### ISSUE TITLE
problem with SqueezeMeta_conf.pl in anvio

### ISSUE THREAD

**Issue by Azimdine** [jtamames/SqueezeMeta#734]
Dear all,
After running SqueezeMeta with a big project, i would analyse my data using anvio interface (i have installed version 8).
after running the script: python3 /pasteur/appa/homes/ahabib/miniconda3/envs/SqueezeMeta/SqueezeMeta/utils/anvio_utils/anvi-load-sqm.py -p Result -o /pasteur/zeus/projets/p02/Biomics/Bioinfo/projet_Ariane/Result/results/anvio --num-threads 12 --min-contig-length 2500 --run-scg-taxonomy --run-HMMS

i get this There must be some problem with "/pasteur/appa/homes/ahabib/miniconda3/envs/SqueezeMeta/SqueezeMeta/utils/anvio_utils/sqm2anvio.pl Result Result/results/sqm2anvio 8.0".
It's better to stop and check it

and when i check the anvio-load-sm.logfile, i have this: Can't find SqueezeMeta_conf.pl in Result
I don't understand because SqueezeMeta_conf.pl is in Result directory

Any suggestions, solutions?
cordially
Azim
[anvi-load-sqm.logfile.zip](https://github.com/jtamames/SqueezeMeta/files/12801876/anvi-load-sqm.logfile.zip)

**Comment by jtamames** [comment]
Hello Azim
I understand that Result is the name of your project. Are you running the command in the upper directory? (I guess that must be projet_Ariane)
Best,
J

**Comment by Azimdine** [comment]
When i had lunch SqueezeMeta, -p was Result. script was SqueezeMeta.pl -m coassembly -p Result -s SN_B6781_1.txt -f B6781_1 --nopfam -b 20 -t 36

datas (fastq, SN_B6781_1.txt ...) was in projet_Ariane directory.
So i thought that -p in anvio was to be Result.

I try with projet_Ariane as -p
cordially
Azim

**Comment by Azimdine** [comment]
i have the same result
Can't find SqueezeMeta_conf.pl in projet_Ariane. Is the project path ok? at /pasteur/appa/homes/ahabib/miniconda3/envs/SqueezeMeta/SqueezeMeta/utils/anvio_utils/sqm2anvio.pl line 21.

**Comment by jtamames** [comment]
Yes, Result is the project, that's clear, but where are you running the anvi-load-sqm.py? You must run it in projet_Ariane directory

**Comment by Azimdine** [comment]
that's working
thank you very much

**Comment by jtamames** [comment]
Great! Remember, yopu must run the script where you ran SqueezeMeta originally, OR, you can specify the full path for the project. That is, -p /pasteur/zeus/projets/p02/Biomics/Bioinfo/projet_Ariane/Result must work every time
Best,
J

**Comment by Azimdine** [comment]
thanks
